pycaret/time_series/forecasting/oop.py

```diff
@@ -1580,6 +1580,10 @@ def compare_models(
 
         """
 
+        if not self._setup_ran:
+            raise Exception("setup() must be ran before all other functions. \
```

Comment:
> "setup() must be ran before all other functions"

This is technically not True. You can still run predict_model on a loaded model without running setup. I would reword this as "This function requires the user to run setup() first."